aiida/common/__init__.py

```diff
@@ -7,7 +7,13 @@
 # For further information on the license, see the LICENSE.txt file        #
 # For further information please visit http://www.aiida.net               #
 ###########################################################################
+"""AiiDA common functionality"""
+
 from __future__ import division
 from __future__ import print_function
 from __future__ import absolute_import
 from aiida.common.log import AIIDA_LOGGER
+
+# pylint: disable=wildcard-import
+
+from .profile import *
```

Comment:
Do we really need a wildcard here?
Perhaps just import the functions we want (I guess the constants defined in profile.py are not needed here (?))

Reply:
Changed this to what we do in other prats of the code (which I borrowed from the asyncio codebase):

    from .profile import *

    __all__ = profile.__all__


This way we delegate what gets imported to `profile` itself.